src/components/cluster/detail/upgrade_cluster_modal.js

```diff
@@ -113,13 +113,15 @@ class UpgradeClusterModal extends React.Component {
           <b>Changes</b>
         </p>
         <ul>
-          {this.props.targetRelease.changelog.map((changelog, i) => {
-            return (
-              <li key={changelog.component + i}>
-                <b>{changelog.component}:</b> {changelog.description}
-              </li>
-            );
-          })}
+          {_.sortBy(this.props.targetRelease.changelog, 'component').map(
+            (changelog, i) => {
+              return (
+                <li key={changelog.component + i}>
+                  <b>{changelog.component}:</b> {changelog.description}
+                </li>
+              );
+            }
+          )}
```

Comment:
@marians LGTM! 

I know that you was just adding sorting, and maybe this comment of mine is a little bit pointless, but I can't help but saying that I would have done it in a more idiomatic way probably, for readibility sake and also because this way we could get rid at some point in the future of underscore or other dependencies:

```
const { changelog } = this.props.targetRelease;
const sortedChangelod = changelog.sort((a, b) => (a.component > b.component) ? 1 : -1);

sortedChangelog.map( ({component, description }) => (
  <li key={component + i}>
    <b>{component}:</b> {description}
  </li>)
```